UnityProject/Assets/Scripts/Medical/HealthScanner.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class HealthScanner : PickUpTrigger
+{
+	public void PlayerFound(PlayerHealth Playerhealth) {
+		string ToShow = (Playerhealth.name + " is " + Playerhealth.ConsciousState.ToString() + "\n"
+			+ "OverallHealth = " + Playerhealth.OverallHealth.ToString() + " Blood level = " + Playerhealth.bloodSystem.BloodLevel.ToString() + "\n"
+						 + "Blood oxygen level = " + Playerhealth.bloodSystem.OxygenLevel.ToString() + "\n");
+		string StringBuffer = "";
+		float TotalBruteDamage = 0;
+		float TotalBurnDamage = 0;
+		float TotalOxygendamage = Playerhealth.CalculateOverallBloodLossDamage();
+		foreach (BodyPartBehaviour BodyPart in Playerhealth.BodyParts)
+		{
+			StringBuffer += BodyPart.Type.ToString() + "\t";
+			StringBuffer += BodyPart.BruteDamage.ToString() + "\t";
+			TotalBruteDamage += BodyPart.BruteDamage;
+			StringBuffer += BodyPart.BurnDamage.ToString();
+			TotalBurnDamage += BodyPart.BurnDamage;
+			StringBuffer += "\n";
+		}
+		ToShow = ToShow + "Overall, Brute " + TotalBruteDamage.ToString() + " Burn " + TotalBurnDamage.ToString() + " OxyLoss " + TotalOxygendamage.ToString() + "\n" + "Body Part, Brute, Burn \n" + StringBuffer;
+		ChatRelay.Instance.AddToChatLogClient(ToShow, ChatChannel.Examine);
```

Comment:
Just realized that everyone knows everyone else's health details. ouch
So this scanner is totally clientside

Reply:
I didn't think about that but It worked in multiplayer